app/util/conf/base_config_loader.py

```diff
@@ -240,6 +240,12 @@ def _cast_and_set(self, key, value, config):
                 value = expanduser(value)
             config.set(key, value)
 
+    def _main_exec_path(self):
```

Comment:
Maybe add a comment to this method about what sys.argv looks like for different execution cases? Is this implementation specific to pex execution?